lib/experiment_sdk_flutter.dart

```diff
@@ -20,9 +21,12 @@ class Experiment {
   }
 
   /// Initialize ExperimentClient with Amplitude instance to handle exposure events
-  static ExperimentClient initializeWithAmplitude(
-      {required String apiKey, ExperimentConfig? config}) {
-    final trackExposureProvider = AnalyticsExposureTrackingProvider();
+  static ExperimentClient initializeWithAmplitude({
```

Comment:
**consider** creating another static method with implicit Amplitude param and deprecation of this one.